jwst/assign_wcs/nircam.py

```diff
@@ -139,7 +139,6 @@ def tsgrism(input_model, reference_files):
         dispx = f.dispx
         dispy = f.dispy
         invdispx = f.invdispx
-        invdispy = f.invdispy
```

Comment:
@sosey Is it OK to remove this?

Reply:
no. it is necessary.

Reply:
It's used for the column grism model in WFSS

Reply:
Oh, I see. But that  is a different function. This one here is `tsgrism` so perhaps can be removed?

Reply:
ah - when I clicked it was taking me to the new file without the line, and only the wfss part...I had missed the grayed out text suggesting it was in tsgrism in the  master...yah, you can remove it. I've already removed it my PR for this mode as well.